.github/workflows/sanity_checks.yml

```diff
@@ -47,6 +47,7 @@ jobs:
   VisualStudio:
     runs-on: windows-latest
     strategy:
+      fail-fast: false
```

Comment:
Temporarily including to make CI verification easier, as pointed out by @benoit-pierre